app/jobs/in_person/enrollments_ready_for_status_check_job.rb

```diff
@@ -0,0 +1,51 @@
+module InPerson
+  # This job checks a queue regularly to determine whether USPS has notitied us
+  # about whether an in-person enrollment is ready to have its status checked. If
+  # the enrollment is ready, then this job updates a flag on the enrollment so that it
+  # will be checked earlier than other enrollments.
+  class EnrollmentsReadyForStatusCheckJob < ApplicationJob
+    include InPerson::EnrollmentsReadyForStatusCheck::UsesAnalytics
+    include InPerson::EnrollmentsReadyForStatusCheck::UsesSqsClient
```

Comment:
Is this inclusion necessary, as BatchProcessor already includes this module?

Reply:
Strictly speaking no, but I consider it important since this class directly uses the `poll` method.